### ISSUE TITLE
"Python error in main script"

### ISSUE THREAD

**Issue by Snargola** [smartavionics/Cura#149]
**Application version**
20211110

**Platform**
Windows 10

**Printer**
N/A

**Reproduction steps**
Started the program

**Screenshot(s)**

![error](https://user-images.githubusercontent.com/70974193/148624812-d09d9ac4-57d4-4b0a-81cb-ec28ec240b10.jpg)

**Actual results**
Program crashes

**Expected results**
Program should start

**Comment by smartavionics** [comment]
Hmm, don't know what the problem is there. How are you starting Cura?

**Comment by Snargola** [comment]
Hello and thank you for your reply.

I have downloaded, and installed, the .exe file from this location:
https://github.com/smartavionics/Cura/releases/download/20211110/Cura-mb-master-win64-20211110.exe

I have accepted all of the defaults.
It created an icon on my Start menu that I have tried to start the program from. I have also tried to start it directly from File Explorer, with no luck. I have tried restarting my computer because, you know, Windows, with the same results.

The error points to a path/user that does not exist on my computer.

I'm going to attempt an uninstall/reinstall but I'm not holding my breath.

Thank you for your time and I look forward to any help you may be able to provide.

**Comment by smartavionics** [comment]
> The error points to a path/user that does not exist on my computer.

Yes, that path is the original path of the file when the app was built. That's OK.

Do you get a cura.log file under app-data/roaming/cura/master ?

**Comment by Snargola** [comment]
This is what I have:
![error2](https://user-images.githubusercontent.com/70974193/148651140-02a7eba3-c4d7-4504-8a94-b8ff6660d60e.jpg)

**Comment by smartavionics** [comment]
Hmm, no master folder. Must be crashing before cura.log is opened.

Do you have stderr.log ?

**Comment by smartavionics** [comment]
That should be in AppData/Roaming/cura

**Comment by smartavionics** [comment]
Yes, you do, I see it in the image above. What's inside that, please?

**Comment by Snargola** [comment]
> Yes, you do, I see it in the image above. What's inside that, please?

You want the whole file? I'm assuming I can drag it into this window...?

**Comment by Snargola** [comment]
[stderr.log](https://github.com/smartavionics/Cura/files/7833534/stderr.log)

**Comment by Snargola** [comment]
For what it's worth, I can't run it as an Administrator, either.

**Comment by smartavionics** [comment]
Thanks for the stderr.log file. Unfortunately, it only contains info from a previous run of 4.12.

So I don't know why my release is not working for you. Nobody else has reported a similar problem. Have you tried the next release, 20201119 ?

**Comment by Snargola** [comment]
> Thanks for the stderr.log file. Unfortunately, it only contains info from a previous run of 4.12.
> 
> So I don't know why my release is not working for you. Nobody else has reported a similar problem. Have you tried the next release, 20201119 ?

I'll try it later today. Thank you for the assistance.

**Comment by Snargola** [comment]
(sigh) Same problem with an older release :(

I am able to compile Marlin. How many orders of magnitude more complex is it to attempt to compile this?

Side note, and I don't think it has anything to do with the problem, here is this:
![error3](https://user-images.githubusercontent.com/70974193/148661096-91389c60-84fc-4587-a2ff-abe716934cd5.jpg)

**Comment by smartavionics** [comment]
Well, I don't know what's going wrong there. Do you have any other python based programs installed on your system? Perhaps there's some conflict with another program or DLLs? Sorry, I'm clueless when it comes to Windows as I never use it apart from when I build Cura releases.

**Comment by Snargola** [comment]
I probably wouldn't know a Python based program if it bit me in the face :) Thank you for your assistance. I'm going to keep messing with it.

**Comment by Snargola** [comment]
I installed Python, because, why not. Didn't help.

I found this, but really have no idea what they are talking about. Maybe you can geek it out?
https://github.com/Ultimaker/cura-build/issues/123

**Comment by joe-ko** [comment]
i get the same errormessage here. only version 20210908 works but not the versions newer

**Comment by smartavionics** [comment]
Well, there's not much difference between 20210908 and the later releases so hopefully you can use that. I will be making a new release soon after 4.13 is released.

**Comment by Snargola** [comment]
> i get the same errormessage here. only version 20210908 works but not the versions newer

20210908, not working for me :(

**Comment by notcraig** [comment]
Any news on this? Running in the same issue. 20210908 and newer.

**Comment by Snargola** [comment]
After trying way too many things, I never got it work :(

**Comment by notcraig** [comment]
> Any news on this? Running in the same issue. 20210908 and newer.

FWIW, this only happens on the Windows version. This isnt an issue with the Linux AppImage
